airbyte-integrations/connectors/source-mssql/src/test/java/io/airbyte/integrations/source/mssql/CdcMssqlSourceTest.java

```diff
@@ -299,41 +282,6 @@ void testAssertCdcSchemaQueryable() {
         () -> source().assertCdcSchemaQueryable(config(), testDatabase()));
   }
 
-  @Test
-  void testAssertSqlServerAgentRunning() {
-    testdb.withAgentStopped().withWaitUntilAgentStopped();
-    // assert expected failure if sql server agent stopped
-    assertThrows(RuntimeException.class, () -> source().assertSqlServerAgentRunning(testDatabase()));
-    // assert success if sql server agent running
-    testdb.withAgentStarted().withWaitUntilAgentRunning();
-    assertDoesNotThrow(() -> source().assertSqlServerAgentRunning(testDatabase()));
-  }
-
-  // Ensure the CDC check operations are included when CDC is enabled
-  // todo: make this better by checking the returned checkOperations from source.getCheckOperations
-  @Test
-  void testCdcCheckOperations() throws Exception {
-    // assertCdcEnabledInDb
-    testdb.withoutCdc();
-    AirbyteConnectionStatus status = source().check(config());
-    assertEquals(status.getStatus(), AirbyteConnectionStatus.Status.FAILED);
-    testdb.withCdc();
-    // assertCdcSchemaQueryable
-    testdb.with("REVOKE SELECT ON SCHEMA :: [cdc] TO %s", testUserName());
-    status = source().check(config());
-    assertEquals(status.getStatus(), AirbyteConnectionStatus.Status.FAILED);
-    testdb.with("GRANT SELECT ON SCHEMA :: [cdc] TO %s", testUserName());
-
-    // assertSqlServerAgentRunning
-
-    testdb.withAgentStopped().withWaitUntilAgentStopped();
-    status = source().check(config());
-    assertEquals(status.getStatus(), AirbyteConnectionStatus.Status.FAILED);
-    testdb.withAgentStarted().withWaitUntilAgentRunning();
-    status = source().check(config());
-    assertEquals(status.getStatus(), AirbyteConnectionStatus.Status.FAILED);
-  }
-
```

Comment:
good riddance

Reply:
that's just moved to another test that'll be isolated. This test class is super slow, so sharing the container and running in parallel would significantly improve the speed of the CI loop. But that's only possible if we don't mess with the container. So putting those 2 tests in a different class makes sense